### PR TITLE
vagrant-service-manager version 0.0.3 release

### LINKED ISSUE #74: vagrant-service-manager plugin version 0.0.3 release
Tracker issue for version 0.0.3 release.
Following are the issues / PR are addressed for the release

Fix #66 
Fix #63 
Fix #59 
Fix #23 
Fix #8 
Fix #45 
Fix #78 

To be merged:
Fix #54

### PR DESCRIPTION
**Pull request by navidshaikh**
Fix #74
- Updates CHANGELOG.md with date of release and change logs
- Updates SPEC file with date of release and change logs
- Fixes the bogus dates of earlier releases in SPEC file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Changelog
 
-## v0.0.3 Feb 25, 2016
+## v0.0.3 Mar 01, 2016
```
Comment (brgnepal) [inline CHANGELOG.md:3]: I think the month name should be `March`.

File: vagrant-service-manager.spec [hunk 2]
```diff
@@ -85,8 +85,10 @@ popd
 %{vagrant_plugin_instdir}/TODO
 
 %changelog
-* Thu Feb 25 2016 Navid Shaikh - 0.0.3-1
+* Thu Mar 01 2016 Navid Shaikh - 0.0.3-1
```
Comment (brgnepal) [inline vagrant-service-manager.spec]: Same here. The month name should be `March`.
And Day name from `Thu` to `Tue`.

Comment (navidshaikh) [inline vagrant-service-manager.spec]: I think Mar is correct.
Check https://fedoraproject.org/wiki/Packaging:Guidelines#Multiple_Changelog_Entries_per_Release

**Comment by brgnepal** [comment]
@navidshaikh : Added comments. Rest looks fine.

**Comment by LalatenduMohanty** [comment]
LGTM. @navidshaikh you should wait till when you are ready with other things  then merge this patch and then do the release.

File: vagrant-service-manager.spec (hunk 2 above)
Comment (brgnepal) [inline vagrant-service-manager.spec]: @bexelbie Your feedback please.

**Comment by navidshaikh** [comment]
> @navidshaikh you should wait till when you are ready with other things then merge this patch and then do the release.

Roger!

**Comment by bexelbie** [comment]
@navidshaikh please rebase and then LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (bexelbie) [inline CHANGELOG.md:3]: So far month has been in MMM form.

**Comment by navidshaikh** [comment]
rebased.

**Pull request merged**